.gitignore

```diff
@@ -1,7 +1,6 @@
 /src/runtime/interopNative.cs
 
 # General binaries and Build results
-*.dll
```

Comment:
This is not something we want.

Reply:
Removing this line did have any effect except making the baseline DLL visible to GIT again. Build outputs are covered by `bin/` and `obj/` patterns.

Reply:
It makes it too easy to commit things by accident. If you really want to commit `Python.Runtime.dll` (which is what I am mainly concerned about since this approach bloats the repo over time), you can just exclude it explicitly here with `!.../Python.Runtime.dll`.